makeservices/easywp

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# Creates symlink to user's webroot. If the symlink exists and files
+# are present in webroot, they are backed up in the user's home directory.
+# A user's database is created or, if it already exists, the password 
+# is reset. WordPress is installed and the wp-config file is filled
+# out with the requisite information. Further configuration can be 
+# done by the user from a browser.  
+#
+# WARNING: This script should generally be run on a account without
+#          an existing website or database!
+#
+
+set -euxo pipefail
+shopt -s dotglob
+
+php_defines() {
+    sed -i "s/$1/$2/" "$WPCONFIG"
+}
+
+WEBROOT=$HOME/public_html
+WPCONFIG=wp-config.php
+USER=$(whoami)
+
+# Create symlink to webroot
+makehttp
+cd "$WEBROOT"
+
+# If user has files, move them to a new location. 
+if [ -n "$(ls -A)" ]; then
+    TS=$(date +"%Y-%m-%d_%H:%M:%S")
+    BACKUP="$HOME/webroot-$TS"
+    mkdir -p "$BACKUP"
+    echo "Webroot not empty. Moving current files to $BACKUP..." 
+    mv ./* "$BACKUP"
+fi
+
+# Get SQL password
+echo "Resetting database..."
```

Comment:
I would change this to `Resetting database password...` so it's clear that only the password is reset.